source/katana/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: home
+sidebar: none
+product: katana
+generator: pagination
+pagination:
+    max_per_page: 5
```

Comment:
This is generating a ton of extra pages. Please make this behave more like the other pages _with_ a sidebar. The sidebar already disappears on smaller screens. It just makes this harder to navigate and maintain.